.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2
+jobs:
+  setup:
+    docker: &docker
+      - image: circleci/ruby:2.5.3-node
```

Comment:
Is node needed? It should work with just `circleci/ruby:2.5.3` as well, right?

Reply:
I tried it, V8 engine needed which comes through node js

```ruby
bundle exec rake rubocop
rake aborted!
ExecJS::RuntimeUnavailable: Could not find a JavaScript runtime. 
```
```ruby
gem 'rubocop-ci', github: 'ad2games/rubocop-ci'
# requires coffee lint :/
```